README.md

```diff
@@ -1,6 +1,45 @@
-# docker-lounge
+<h1 align="center">
+	<img
+		width="300"
+		alt="The Lounge"
+		src="https://raw.githubusercontent.com/thelounge/lounge/b637f50522cea5c666402507582b5e945a733e96/client/img/logo-vertical-transparent-bg.svg">
+</h1>
 
-Docker container for [The Lounge](https://thelounge.github.io/), a self-hosted web IRC client.
+<h3 align="center">
+	Docker container for The Lounge, a self-hosted web IRC client.
+</h2>
```

Comment:
`<h3>`/`</h2>`. I realize I did the same thing in my PR :man_facepalming:

Reply:
Make this `h3` and not `h2`, as `h2` adds another line on top of the one above. Result is the screenshots we already have on the main repo's PR, as they were all treated as `h3`.